single.php

```diff
@@ -1,32 +1,50 @@
 <?php
 /**
- * The Template for displaying all single posts.
  *
- * @package MIT_Libraries_Parent
- * @since 1.2.1
+ * This is the template that displays single posts
+ *
+ * @package WordPress
+ * @subpackage Twenty_Twelve
+ * @since Twenty Twelve 1.0
  */
+ 
+$pageRoot = getRoot($post);
+$section = get_post($pageRoot);
+$isRoot = $section->ID == $post->ID;
 
-get_header(); ?>
-
-	<div id="primary" class="site-content">
-		<div id="content" role="main">
-
-			<?php while ( have_posts() ) : the_post(); ?>
-
-				<?php get_template_part( 'content', get_post_format() ); ?>
 
-				<nav class="nav-single">
-					<h3 class="assistive-text"><?php _e( 'Post navigation', 'twentytwelve' ); ?></h3>
-					<span class="nav-previous"><?php previous_post_link( '%link', '<span class="meta-nav">' . _x( '&larr;', 'Previous post link', 'twentytwelve' ) . '</span> %title' ); ?></span>
-					<span class="nav-next"><?php next_post_link( '%link', '%title <span class="meta-nav">' . _x( '&rarr;', 'Next post link', 'twentytwelve' ) . '</span>' ); ?></span>
-				</nav><!-- .nav-single -->
-
-				<?php comments_template( '', true ); ?>
-
-			<?php endwhile; // end of the loop. ?>
-
-		</div><!-- #content -->
-	</div><!-- #primary -->
+get_header(); ?>
 
-<?php get_sidebar(); ?>
-<?php get_footer(); ?>
+<?php get_template_part('inc/breadcrumbs', 'child') ?>
+	
+<div id="stage" class="inner" role="main">
+
+	<?php get_template_part('inc/postHead'); ?>
```

Comment:
postHead was added here because it is inherited by Child sites which require it.